jsonschema-hyperschema.xml

```diff
@@ -210,6 +184,17 @@
         </section>
 
         <section title="Schema keywords">
+            <t>
+                Hyper-schema keywords can be applied when the instance is valid against
+                the schema that includes those keywords, as outlined in
+                <xref target="json-schema-validation">Section 10.1 of JSON Schema validation</xref>.
+            </t>
+            <t>
+                When multiple subschemas are applicable to a given sub-instance, all "link"
+                arrays MUST be concatenated, in any order, into a single array.  Each object
+                in the resulting array MUST retain its own list of applicable "base" values
+                from the same schema and any parent schemas.
```

Comment:
This paragraph makes me think back about #124. So if links was an object, what does concatenation would mean?
Might be an argument in favor of keeping the array...

Reply:
If `links` becomes an object, then it's an object of arrays, so for each relation that appears in multiple subschemas, you would concatenate the arrays for that relation to produce the array in the resulting combined value.

Reply:
True. I realized about that after writing the comment.